### PR TITLE
Survey Form.

### PR DESCRIPTION
**Pull request by m00h0**

### CODE REVIEW

**Comment by m00h0** [comment]
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



To Highlight 🥇
✅ Descriptive Pull Request title and summary. 
✅ Good use of Github flow 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

Changes Required
 Please check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏


- the link to your image is broken can you fix it please.

- you need linter files in your project so you can check linters.

- your pull request wasn't a professional title and description.

you haven't  GitHub flow method used.

The project isn't professionally documented.

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @AhmedMh0 , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] Reviews given. ✔️ 



## Required Changes ♻️
_Check the comments under the review._
- [ ] Please use GitHub inline comments to perform the review. This means you should write in the line that you request the code to be changed/ improved. Kindly take a look at this [guide](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) and apply the concepts therein to your code reviews.

- [ ] Please make sure your comments are clear, polite and offer reasons why you what the code changed.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -10,7 +10,7 @@ body {
       100deg,rgba(20,20,60,0.2),
      rgba(30,30,50,0.3)
     ),
-    url(https://avic.uobaghdad.edu.iq/wp-content/uploads/sites/54/2020/11/%D8%B5%D9%88%D8%B1%D8%A9-%D9%84%D9%8A%D9%84%D9%8A%D8%A9.png);
+    url(https://avic.uobaghdad.edu.iq/wp-content/uploads/sites/54/2020/11/%D8%B5%D9%88%D8%B1%D8%A9-%D9%84%D9%8A%D9%84%D9%8A%D8%A3.png);
```
Comment (m00h0) [inline style.css:13]: Your link to the image is broken try to change it.

**Review by m00h0 — COMMENTED** [review]
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🥇
✅ Descriptive Pull Request title and summary.
✅ Good use of Github flow
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

Changes Required
Please check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

-  [ ]  the link to your image is broken can you fix it please, because your link of images it doesn't work so replace the link to valid link 

-  [ ]  you need linter files in your project so you can check linters, so you need to add linters files from this link (https://github.com/microverseinc/linters-config/tree/master/html-css).

-  [ ]  your pull request wasn't a professional title and description, you should add some information about what you doing in your project.

- [ ] you haven't GitHub flow method used. you need to make  GitHub files path inside your project 

- [ ]  Please check the comments under the review

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @AhmedMh0  ) in your question so I can receive the notification.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @AhmedMh0, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost

## Suggested changes :recycle: 
- [ ] As requested by the previous reviewer, Kindly make sure your comments are polite and give a reason why you suggested the changes

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Strangeal — APPROVED** [review]
Hi @AhmedMh0, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**